netdata/netdata
# expose CmdLine in apps function

##### Summary

Requested [on Discord](https://discord.com/channels/847502280503590932/1123360878285312101/1123360878285312101).

Disabled by default. Needs the `with-function-cmdline` command option.


##### Test Plan

- Add `with-function-cmdline` to apps's command options.
- Check the "Functions" tab. This requires changing "CmdLine" to any other value because there is a hard-coded rule to hide this column from the "Edit columns" menu (will be fixed tomorrow).

##### Additional Information


<details> <summary>For users: How does this change affect me?</summary>
  
</details>
